go.mod

```diff
@@ -139,6 +139,7 @@ require (
 )
 
 replace (
+	bazil.org/fuse => bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898
```

Comment:
I'm surprised this didn't show up as a new dep in depstats... it's not in our dependency tree currently, right? what is pulling this in?

Reply:
That's what I was wondering too after just noticing... No I don't believe so, I'll go on a hunt in a second.

Reply:
answering my own question, github.com/containerd/continuity pulls it in, and did even on the version we were on before (https://github.com/containerd/continuity/blob/aaeac12a7ffcd198ae25440a9dff125c2e2703a7/continuityfs/fuse.go) but did not have a go.mod file in that version so it didn't show up in the graph

Reply:
Welp, checked back in to answer and it's already checked in 😂. Glad you found it